### PR TITLE
add change links for more than one claimant

### PR DESCRIPTION
**Pull request by sabahirfan**
This PR adds change links for the multiple claimants.

### CODE REVIEW

File: src/main/features/claim/routes/claimant-add.ts [hunk 1]
```diff
@@ -51,6 +51,8 @@ export default express.Router()
           Claimants.addClaimant(res)
           await ClaimDraftMiddleware.save(res, next)
           res.redirect(Paths.claimantTypePage.uri)
+        } else if (res.locals.user.legalClaimDraft.defendants.length > 1) {
```
Comment (dharmendrak) [inline src/main/features/claim/routes/claimant-add.ts]: u missed merging master here, it should be claimants not defendants

Comment (dharmendrak) [inline src/main/features/claim/routes/claimant-add.ts]: what is this step for?? is it requirement??

Comment (sabahirfan) [reply src/main/features/claim/routes/claimant-add.ts]: If we are coming from the details summary page, and we have multiple defendants, then the next page should be defendant-add where we have the table of defendants.

Comment (dharmendrak) [reply src/main/features/claim/routes/claimant-add.ts]: or should have displayed last defendant details?? was this requirement?

Comment (sabahirfan) [reply src/main/features/claim/routes/claimant-add.ts]: displaying last defendant and then going to defendant table does not make any sense.

**Pull request merged**